@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -78,7 +112,22 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
     },
   },
   ({ scale, state }) => (
-    <div ref={state.container} style={{ margin: 'auto', height: `${scale}%`, width: `${scale}%` }} />
+    <>
+      {state.rfb !== undefined && !state.rfbConnected && (
+        <p>
+          <FormattedMessage id='reconnectionAttempt' />
+        </p>
+      )}
+      <div
+        ref={state.container}
+        style={{
+          display: `${state.rfbConnected ? 'block' : 'none'}`,
+          margin: 'auto',
+          height: `${scale}%`,
+          width: `${scale}%`,
+        }}
```

Comment:
Please use styled components now. In another PR if you don't want to drift away from the goal of this one.